src/pudl/output/eia923.py

```diff
@@ -246,6 +293,8 @@ def fuel_receipts_costs_eia923(pudl_engine, freq=None,
             'total_mercury_content': pudl.helpers.sum_na,
             'total_moisture_content': pudl.helpers.sum_na,
             'total_chlorine_content': pudl.helpers.sum_na,
+            'fuel_cost_from_eiaapi': 'any',
+            'fuel_cost_per_unit': 'mean'
```

Comment:
I think just using `mean` here might be incorrect -- wouldn't this need to be a weighted average, weighted by the number of units? Not an issue if we're going to remove the column, as mentioned above.

Reply:
for sure. as i said above, this was just a debugging column. I will remove it.